docs/terminal/atma.md

```diff
@@ -70,17 +70,17 @@ As track-shortening direct to `TAPAT` or `BOGUN` can interfere with arrivals wit
 
 ### Short-Sector STARs
 
-ATMA is responsible for issuing STAR clearances for aircraft bound for NZHN, NZTG, NZRO and NZNP, and shall do so without coordination. 
+ATMA is responsible for issuing STAR clearances for aircraft bound for NZHN and NZTG and shall do so without coordination. 
 
 ### Assigned Heading Departures
 
 Assigned Heading departures may be requested by Tower or ATMA in order to allow for more efficient traffic flow from the runway. Controllers can use this method to ensure that a 30° divergence exists between tracks - either keeping the go-around path clear, or by moving an aircraft away from another aircraft's departure path. These shall be issued in accordance with [NZAA - Assigned Headings](../aerodromes/Class-C/nzaa.md#assigned-headings).
 
-Due to the early turn required, assigned heading departures shall be given to prop aircraft only.
```

Comment:
Please leave this in, as this is the case irl. It's sometimes different for Jet vs Jet successives, but it's best if we simplify it.